app/concepts/halls/use_cases/create_seats.rb

```diff
@@ -9,10 +9,10 @@ def initialize(params:)
         @params = params
       end
 
-      def call(rows, columns)
+      def call(params: @params)
```

Comment:
When you initialize with params, you don't have to second time pass those parameters. If you defined it like[ a instance variable](https://www.rubyguides.com/2019/07/ruby-instance-variables/) (`@` sign, for instance `@params`), you can just use it everywhere in class.